search.scss

```diff
@@ -11,13 +11,13 @@
 		&:hover,
 		&.vui-input-hover,
 		&.vui-input-focus {
-			padding-right: calc(2.2rem - 1px);
+			padding-right: calc(2.1rem - 1px);
```

Comment:
Changes in here are so that there's 9px (0.45rem) between the search icon and the right edge + 9px (0.45rem) between the left edge of the icon and the text + 24px (1.2rem) for the icon itself. 0.45 + 0.45 + 1.2 = 2.1rem.